src/cart/crt_init.c

```diff
@@ -539,7 +549,17 @@ prov_settings_apply(bool primary, crt_provider_t prov, crt_init_options_t *opt)
 	}
 
 
-	return rc;
+	if (prov == CRT_PROV_OFI_CXI) {
+		apply_if_not_set("FI_CXI_OPTIMIZED_MRS", "0");
+		apply_if_not_set("FI_CXI_RX_MATCH_MODE", "hybrid");
+		apply_if_not_set("FI_MR_CACHE_MONITOR", "memhooks");
+		apply_if_not_set("FI_CXI_REQ_BUF_MIN_POSTED", "8");
+		apply_if_not_set("FI_CXI_REQ_BUF_SIZE", "8388608");
+		apply_if_not_set("FI_CXI_DEFAULT_SQ_SIZE", "131072");
+		apply_if_not_set("FI_CXI_OFLOW_BUF_SIZE", "8388608");
+	}
```

Comment:
I don't think we should apply those FI_CXI env variables. `FI_CXI_OPTIMIZED_MRS` is no longer required with newer slingshot, hybrid is the default afaik and it's unclear right now if we should use memhooks or not. The rest of the envs should not be set. Imo only the crt mr cache one should be set. We could eventually add the envs that control the FI CXI CQ fill percent / FI CXI CQ size but that's all really that should be needed.

Reply:
ok, changed to only do mrc enable for cxi as a default